cloud/kubernetes/helm/yugabyte/templates/service.yaml

```diff
@@ -23,27 +23,34 @@ spec:
   selector:
     app: "{{ .label }}"
 
-{{- if .hasLoadBalancer }}
 ---
+{{- $service := . -}}
```

Comment:
Do we unconditionally create this? We should have a way of enabling/disabling this based on a flag. Ok to default to "enabled" in yw. Here maybe make it "disabled" default to make minikube deploys easier.

Reply:
Lets make all the three lbs separate flags in the helm chart, thats probably the safest. But lets always create the UI lbs by default - by setting that flag to true. We may have to discover if folks want lbs or node-port services in the context of various users.